CHANGELOG.md

```diff
@@ -7,7 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-- Add `ec2:ReplaceRoute` permissions to the CAPA controller role
+### Added
+
+- Add `ec2:ReplaceRoute` permissions to the CAPA controller role.
```

Comment:
can we add the note that new CAPA version requires it?

Reply:
Added note about it, PTAL.

Reply:
why only in 1 bullet point not the other? :D

Reply:
That was added in another PR. I'll try to figure out why is needed.

Reply:
oh you approved and the branch got removed :sweat_smile: